app/src/androidTest/java/com/android/voyageur/ui/overview/AddTripTest.kt

```diff
@@ -350,7 +339,81 @@ class AddTripScreenTest {
 
     // Simulate failed image cropping
     composeTestRule.onNodeWithText("Select Image from Gallery").performClick()
-
-    // Verify error toast is shown (would need to check for Toast message)
   }
+  /*@Test
+  fun checkPermissionReturnsTrue() {
+    // Mock context
+    val context = mock(Context::class.java)
+
+    // Mock the permission check to return PERMISSION_GRANTED
+    `when`(ContextCompat.checkSelfPermission(context, "android.permission.READ_MEDIA_IMAGES"))
+        .thenReturn(PackageManager.PERMISSION_GRANTED)
+
+    val result = checkFullPermission(context)
+    assertTrue(result) // Expecting true because permission is granted
+  }*/
+
+  /*@Test
+  fun checkPermissionReturnsTrueLimitedPermission() {
```

Comment:
Are these tests not passing anymore, or why are they commented?

Reply:
I merged what I found when trying to merge with main. I don't know why they're commented

Reply:
It's my bad, I think I left them commented when I moved them to PermissionButtonGalleryTest and they appeared as changes in your PR. I'm sorry for this, could you please delete them? Thanks!